style.css

```diff
@@ -174,6 +174,24 @@ nav {
   border-radius: 10px;
 }
 
+.section-odd .image {
+  transition: 1s;
+}
+
+.section-even .image {
```

Comment:
- Good job on your accessibility report, however, it lacks some report requirements listed below:
  
 **No report on**
     - Page title [How to check for Page titles](https://www.w3.org/WAI/test-evaluate/preliminary/#title)
     - image text alternatives `(<img alt="alt text ">)` [How to check Image text Alt](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
    - text headings [How to check text heading](https://www.w3.org/WAI/test-evaluate/preliminary/#headings)

**The report  should include a statement pertaining to every point checked:**
- If everything was ok,  make a statement about that (e.g. "Page titles - no issues found").
- If issues were found, Your report should have a  link to the corresponding line of code, in the Pull Request changes